images/tier1/google-drive.svg

```diff
@@ -1,3 +1,5 @@
-<svg xmlns="http://www.w3.org/2000/svg" height="18" width="18" viewBox="0 0 18 18">
-  <path fill="#565a5c" d="M17.98 11.67c.012.068.018.142.02.216 0-.072-.004-.145-.02-.216zM3.702 18H14.74c.284 0 .54-.217.666-.487l2.11-4.51H6.212L3.702 18zM16.2 11.003h1.7L12.532.213c-.128-.27-.383-.206-.66-.206H6.65l6.004 10.996H16.2zM5.304.778l-.322.662-4.89 10.04c-.12.238-.12.524-.01.762l2.3 4.86 5.704-11.03L5.64 1.417l-.336-.64zM17.98 11.67c-.016-.08-.038-.153-.07-.213h-.01l.01.025c.032.06.054.123.07.188z"/>
-</svg>
```

Comment:
I've never seen `polygon` used before in our SVGs. If I put this through [SVGOMG](https://jakearchibald.github.io/svgomg/), I get this simplified version, which seems equivalent:
```
<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" viewBox="0 0 18 18">
  <path fill="#565a5c" d="M8.68 6.75l-5.57 9.41L0 10.86l5.54-9.57 3.14 5.46zM17.66 10.27h-6.3L5.94.85h6.26l5.46 9.42zM18 10.86l-3.12 5.32H3.76l3.16-5.34h11.07l.01.02z"/>
</svg>
```